english/animal/feed-cat-hot-dog/hot-dog.md

```diff
@@ -0,0 +1,9 @@
+The cat sniffs eagerly at your hot dog, and then snatches it from your fingers. It swiftly disappears down his gullet. 
+
+Nothing happens. You get bored and look around the room again.
+
+When you look back, something has changed. And keeps changing. With disturbing rapidity.
+
+There's no doubt about it. The cat is growing larger.
+
+After about a minute, his head is almost level with your waist. Or - perhaps more accurately - you appear, to him, to have shrunk to a more ... interesting size.  
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.